[2024-09-24T00:39:05.673+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion_15min.trigger_stage_processing manual__2024-09-24T00:38:49.191449+00:00 [queued]>
[2024-09-24T00:39:05.683+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion_15min.trigger_stage_processing manual__2024-09-24T00:38:49.191449+00:00 [queued]>
[2024-09-24T00:39:05.684+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-09-24T00:39:05.696+0000] {taskinstance.py:1382} INFO - Executing <Task(TriggerDagRunOperator): trigger_stage_processing> on 2024-09-24 00:38:49.191449+00:00
[2024-09-24T00:39:05.701+0000] {standard_task_runner.py:57} INFO - Started process 2576 to run task
[2024-09-24T00:39:05.704+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion_15min', 'trigger_stage_processing', 'manual__2024-09-24T00:38:49.191449+00:00', '--job-id', '560', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion_15min.py', '--cfg-path', '/tmp/tmpw97_wo3y']
[2024-09-24T00:39:05.705+0000] {standard_task_runner.py:85} INFO - Job 560: Subtask trigger_stage_processing
[2024-09-24T00:39:05.815+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion_15min.trigger_stage_processing manual__2024-09-24T00:38:49.191449+00:00 [running]> on host airflow-worker
[2024-09-24T00:39:06.000+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion_15min' AIRFLOW_CTX_TASK_ID='trigger_stage_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:38:49.191449+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-24T00:38:49.191449+00:00'
[2024-09-24T00:39:06.093+0000] {trigger_dagrun.py:173} INFO - Clearing stage_processing on 2024-09-24T00:00:00+00:00
[2024-09-24T00:39:06.269+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:39:16.319+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:39:26.379+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:39:36.421+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:39:46.501+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/trigger_dagrun.py", line 220, in execute
    raise AirflowException(f"{self.trigger_dag_id} failed with failed states {state}")
airflow.exceptions.AirflowException: stage_processing failed with failed states failed
[2024-09-24T00:39:46.505+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion_15min, task_id=trigger_stage_processing, execution_date=20240924T003849, start_date=20240924T003905, end_date=20240924T003946
[2024-09-24T00:39:46.526+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 560 for task trigger_stage_processing (stage_processing failed with failed states failed; 2576)
[2024-09-24T00:39:46.555+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-24T00:39:46.600+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
